test/e2e/specs/wp-my-home-support-search-spec.js

```diff
@@ -78,46 +56,32 @@ describe( `[${ host }] My Home "Get help" support search card: (${ screenSize })
 		);
 	} );
 
-	step( 'Verify Inline Help support search is not displayed on My Home', async function () {
-		// The toggle only hides once the "Get help" card is rendered so
-		// we need to give it a little time to be removed.
-		await inlineHelpPopoverComponent.waitForToggleNotToBePresent();
-
-		// Once removed we can assert is is invisible.
-		const isToggleVisible = await inlineHelpPopoverComponent.isToggleVisible();
-		assert.equal(
-			isToggleVisible,
-			false,
-			'Inline Help support search was not correctly hidden on Home page.'
-		);
-	} );
-
```

Comment:
Same here. We need this test. Not sure why it's removed.

Reply:
Moved these FAB tests over to `test/e2e/specs/wp-inline-help-support-search-spec.js ` since it felt more relevant there.